### ISSUE TITLE
[Accessibility] ACC-264.1(Level A) - Select box selected option announced twice

### ISSUE THREAD

**Issue by harshitha-s-kanchan** [SAP/fundamental-ngx#11482]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
When option in the select box is selected using arrow keys in linear reading mode, selected option is read out twice which is incorrect. Option should be read out once.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular - 15
NGX - 0.43.21 (Issue also present in the latest version)
component - `<fd-select>`
Screen reader - JAWS 2023

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
When accessing select box in linear reading mode, press arrow down/up key to select any option,
Below is the screen recording,

https://github.com/SAP/fundamental-ngx/assets/149040487/cfd7f655-793b-4723-92ae-c8612118bf7c

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes the issue is raised by Central Accessibility team
